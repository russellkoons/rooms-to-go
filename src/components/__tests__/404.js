import React from "react"
import renderer from "react-test-renderer"

import NotFoundPage from "../../pages/404"

describe("NotFoundPage", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<NotFoundPage />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})