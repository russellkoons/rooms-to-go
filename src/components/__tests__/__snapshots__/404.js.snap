// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
Array [
  <header
    style={
      Object {
        "background": "rebeccapurple",
        "marginBottom": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "0 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
            "position": "relative",
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          RTG
        </a>
        <a
          href="/cart"
          style={
            Object {
              "color": "white",
              "float": "right",
              "fontSize": "1rem",
              "fontWeight": "700",
              "position": "absolute",
              "right": "1.0875rem",
              "textDecoration": "none",
              "top": "1.5rem",
            }
          }
        >
          CART
        </a>
      </h1>
    </div>
  </header>,
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "0px 1.0875rem 1.45rem",
        "paddingTop": 0,
      }
    }
  >
    <main>
      <h1
        cart={Array []}
        setCart={[Function]}
      >
        NOT FOUND
      </h1>
      <p
        cart={Array []}
        setCart={[Function]}
      >
        You just hit a route that doesn't exist... the sadness.
      </p>
    </main>
    <footer />
  </div>,
]
`;
